src/zenml/entrypoints/entrypoint.py

```diff
@@ -39,12 +40,23 @@ def main() -> None:
     # is not wrapped in a function or an `if __name__== "__main__":` check)
     constants.SHOULD_PREVENT_PIPELINE_EXECUTION = True
 
+    source_utils.set_custom_source_root(source_root="custom_source_root")
```

Comment:
Why is this needed?

Reply:
because we are running from notebook (databricks apparently runs the code from notebook env) so it detect that there is no `zenml init` so setting a custom root is a solution that worked